### DISCUSSION TITLE
URGENT! How to Find Cache Origins?

### DISCUSSION THREAD

**Discussion by Corrosive1** [mastodon/mastodon#31588] [Q&A]
Hello, so heres what i did

1) Downloaded the folder home\mastodon\live\public\system\*
2) found a few Pictures from other Domains that should NOT be there cuz they are likely not legal
3) files are under something like "home\mastodon\live\public\system\cache\media_attachments\files.\.\.\."

now i got the Filename something like 816abc7078f5b476.PNG 

i logged into the PostgreSQL Database and checked the media_attachments Database, but it does not find the Filename, only the Filenames of files uploaded on MY Instance

1) How do i find the files and from which instance the File is coming from?

Thanks

**Comment by Gargron** [comment]
The path to the file (001/234/567/...) is the numeric ID of the media attachment.

**Reply by Corrosive1** [reply]
may i ak what this means please? im just trying to find out from which instance the file comes from.

**Reply by Corrosive1** [reply]
edit: i tried to take that ID (in this case) 112603415192238609and search the ID of the PostgreSQL database for it using a Webmin Interface, but i only get 

`SQL select count(*) from "media_attachments" where "account_id" like '%112603415192238609%' failed : ERROR: operator does not exist: bigint ~~ unknown LINE 1: ...nt(*) from "media_attachments" where "account_id" like '%112... ^ HINT: No operator matches the given name and argument types. You might need to add explicit type casts.`
